jboss-kie-kieserver/tests/bats/resources/standalone/configuration/standalone-openshift.xml

```diff
@@ -0,0 +1,695 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there's no need to provide a full xmlk file for the bats tests, just the subsystem  you're going to test should be enough.
Even better, do not add a new file, copy the standalone-openshift.xml file from here: https://github.com/jboss-container-images/jboss-kie-modules/tree/master/jboss-eap-config-openshift/EAP7.3.0/added

Reply:
ok